### PR TITLE
Reordered Hamburger Menu with layout mentioned in #1893

### PR DESCRIPTION
**Pull request by rohanthacker**
- [ x ] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [ x ] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ x ] Ran `git rebase -i` to squash commits if needed.

Original Issue #1893

### CODE REVIEW

File: js/contextMenus.js [hunk 1]
```diff
@@ -578,26 +569,14 @@ function hamburgerTemplateInit (location, e) {
         CommonMenu.separatorMenuItem,
         CommonMenu.importBookmarksMenuItem()
       ]
-    }, {
-      label: locale.translation('bravery'),
-      submenu: [
```
Comment (bbondy) [inline js/contextMenus.js]: we should still prefer the Bravery menu. 

CC @bradleyrichter which order would you like it in?

File: js/contextMenus.js [hunk 2]
```diff
@@ -578,26 +569,14 @@ function hamburgerTemplateInit (location, e) {
...
     CommonMenu.downloadsMenuItem(),
+    CommonMenu.findOnPageMenuItem(),
+    CommonMenu.printMenuItem(),
     CommonMenu.separatorMenuItem,
-    {
-      label: locale.translation('help'),
-      submenu: [
-        CommonMenu.aboutBraveMenuItem(),
-        CommonMenu.separatorMenuItem,
-        CommonMenu.checkForUpdateMenuItem(),
```
Comment (bbondy) [inline js/contextMenus.js]: Ditto I don't think we should get rid of about brave and check for updates.
CC @bradleyrichter

File: js/contextMenus.js [hunk 3]
```diff
@@ -578,26 +569,14 @@ function hamburgerTemplateInit (location, e) {
...
-        CommonMenu.separatorMenuItem,
-        CommonMenu.reportAnIssueMenuItem(),
-        CommonMenu.submitFeedbackMenuItem()
```
Comment (bbondy) [inline js/contextMenus.js]: @brad do you want report an issue and submit feedback removed now?

Comment (bradleyrichter) [inline js/contextMenus.js]: We should keep one of them. It promotes teamwork.  :   )

Chrome has Report and Issue. FX has Submit feedback. Any pref @bbondy ?

**Comment by rohanthacker** [comment]
@bradleyrichter @bbondy . The original menu items have been added back in pending appropriate Order, and decision on Report an Issue and Submit Feedback.

File: js/contextMenus.js (hunk 3 above)
Comment (rohanthacker) [inline js/contextMenus.js]: @bbondy @bradleyrichter  I believe I misinterpreted the ticket, My mistake, excuse that

File: js/components/urlBar.js [hunk 4]
```diff
@@ -368,6 +369,14 @@ class UrlBar extends ImmutableComponent {
       action='#'
       id='urlbar'
       ref='urlbar'>
+      <Button iconClass={this.titleMode ? 'fa-star' : 'fa-star-o'}
```
Comment (bbondy) [inline js/components/urlBar.js]: Not sure I understand, why was bookmarks moved into the urlbar component? It seems unrelated to the rest of the changeset.

Comment (rohanthacker) [inline js/components/urlBar.js]: It is, It should be on another branch, not sure how it slipped in. This change is to tackle the new bookmark design.

**Comment by bbondy** [comment]
Sorry for the delay, thank you! Merging in!

**Comment by rohanthacker** [comment]
👍
